Content.Shared/Whitelist/EntityWhitelistSystem.cs

```diff
@@ -11,12 +12,14 @@ public sealed class EntityWhitelistSystem : EntitySystem
     [Dependency] private readonly SharedRoleSystem _roles = default!;
     [Dependency] private readonly TagSystem _tag = default!;
 
+    private ISawmill _sawmill = default!; // DeltaV
```

Comment:
its a system just use Log.Debug

Reply:
won't Log.Debug kill performance when doing a ton of terminal IOl? i'm not sure if it's actually more expensive compared to a single sawmill, but that is what's been happening whenever we got invalid component or tag names

Reply:
update: it kills TPS just as much :trollface: